doc/guides/device_mgmt/smp_groups/smp_group_1.rst

```diff
@@ -0,0 +1,358 @@
+    | ``1``         | Image upload                                  |
+    +---------------+-----------------------------------------------+
+    | ``2``         | File                                          |
+    |               | (reserved but not supported by Zephyr)        |
+    +---------------+-----------------------------------------------+
+    | ``3``         | Corelist                                      |
+    |               | (reserved but not supported by Zephyr)        |
+    +---------------+-----------------------------------------------+
+    | ``4``         | Coreload                                      |
+    |               | (reserved but not supported by Zephyr)        |
+    +---------------+-----------------------------------------------+
+    | ``5``         | Image erase                                   |
+    +---------------+-----------------------------------------------+
+
+State of images
+***************
+
+The command is used to set state of images and obtain list of images
+with their current state.
+
+Get state of images request
+===========================
+
+Get state of images request header fields
+
+.. table::
+    :align: center
+
+    +--------+--------------+----------------+
+    | ``OP`` | ``Group ID`` | ``Command ID`` |
+    +========+==============+================+
+    | ``0``  | ``1``        |  ``0``         |
+    +--------+--------------+----------------+
+
+The command does not send any payload in request, or sends empty CBOR map.
+
+.. _mcumgr_smp_protocol_op_1_grp_1_cmd_0:
+
+Get state of images response
+============================
+
+Get state of images response header fields
+
+.. table::
+    :align: center
+
+    +--------+--------------+----------------+
+    | ``OP`` | ``Group ID`` | ``Command ID`` |
+    +========+==============+================+
+    | ``1``  | ``1``        |  ``0``         |
+    +--------+--------------+----------------+
+
+.. note::
+    Below definition of the response contains "image" field that has been marked
+    as optional(opt): the field may not appear in response when target application
+    does not support more than one image.  The field is mandatory when application
+    supports more than one application image to allow identifying which image
+    information is listed.
+
+A response will only contain information for valid images, if an image can not
+be identified as valid it is simply skipped.
+
+CBOR Payload of response command:
+
+.. code-block:: none
+
+    {
+        (str)"images" : [
+            {
+                (str,opt)"image"    : (int)
+                (str)"slot"         : (int)
+                (str)"version"      : (str)
+                (str)"hash"         ; (str)
+                (str)"bootable"     : (bool)
+                (str)"pending"      : (bool)
+                (str)"confirmed"    : (bool)
+                (str)"active"       : (bool)
+                (str)"permanent"    : (bool)
```

Comment:
Yes, but I want to avoid listing Kconfig here, in protocol specification, The person implementing, for example a client, will not care about Kconfig options in Zephyr and these are separately documented anyway.